lib/extensions/reset_embroidery_settings.py

```diff
@@ -0,0 +1,47 @@
+import inkex
+
+from .base import InkstitchExtension
+
+
+class ResetEmbroiderySettings(InkstitchExtension):
+    def effect(self):
+        # TODO: Extension does not work for ignored layers/objects
```

Comment:
I think we can support ignored layers/objects if we work with `self.selected` directly.  Skip the `self.get_elements()` and do `for node in self.selected` instead.  We might have to deal with groups by going through all of their children, too.

Another good reason to avoid using `self.get_elements()`: that way this extension will work even if they have invalid paths.  Ink/Stitch elements will never get created, so we won't try to run `is_valid` on them.  That way people can clear the Ink/Stitch attributes even when Ink/Stitch thinks elements aren't valid, like @tyrosinase's satin-bean-thing.